src/app/app-routing.module.ts

```diff
@@ -44,6 +44,15 @@ export const routes: Routes = [
       title: 'Verify'
     }
   },
+  
```

Comment:
tslint is failing here due to trailing whitespace

Reply:
@jiekang Fixed. Used Github UI for resolving conflicts which is why this typo happened.